### ISSUE TITLE
ES6 Support

### ISSUE THREAD

**Issue by darthtrevino** [alexeyraspopov/gulp-complexity#5]
I'm using Babel and ES6, and complexity analysis doesn't work on my code base. It would be nice if the analysis supported ES6. I think you can get this by using an esprima fork (e.g. esprima-fb).

**Comment by alexeyraspopov** [comment]
Hi @darthtrevino! I'll try to investigate capabilities asap. Out of the box, [complexity-report](https://github.com/philbooth/complexity-report) doesn't have ES6 support. And it's the main issue.

**Comment by alexeyraspopov** [comment]
[Well...](https://github.com/philbooth/escomplex/issues/13#issuecomment-118313951)

**Comment by daedalus28** [comment]
The official fork of escomplex by jared-stilwell seems to support es6 (and also is the official npm package now): 
https://github.com/jared-stilwell/escomplex.
https://github.com/jared-stilwell/complexity-report

It seems like a version bump of the npm package would be sufficient.

**Comment by rakeshmenon** [comment]
Right.. Also, quite a few options are not available in this version of complexity-report

**Comment by kadishmal** [comment]
Any news on this?

**Comment by alexeyraspopov** [comment]
IDK when I can find time for this, so PRs welcome.

**Comment by jreading** [comment]
I might take a stab at this.
